playground/routes/error-route.ts

```diff
@@ -0,0 +1,9 @@
+export default defineCachedEventHandler(
```

Comment:
We sould probably remove the test route from the playground file 😋

Reply:
Done! Thanks for the suggestion